Aztec/Classes/TextKit/TextView.swift

```diff
@@ -674,17 +679,19 @@ open class TextView: UITextView {
         }
 
         let isBegginingOfListItem = storage.isStartOfNewLine(atLocation: range.location)
+        guard text == "\n" && beforeString == "\n" && afterString == "\n" && isBegginingOfListItem else {
```

Comment:
If it's okay with you, let's, instead, delay refactoring the `refreshAfterInsertion` / `refreshAfterDeletion` methods.

We may (_may_) have a master fix for all of the weirdness'ess we've been seeing around, and it would definitely involve some extra work in those precise methods.

Thanks!!!